modules/controller_utils/tests.py

```diff
@@ -30,7 +30,7 @@ def fake_tla() -> str:
 
 @contextlib.contextmanager
 def mock_match_file() -> Iterator[IO[str]]:
-    with tempfile.NamedTemporaryFile(suffix='.json', mode='r+t') as f:
+    with tempfile.NamedTemporaryFile(suffix='.json', mode='r+t', delete=False) as f:
```

Comment:
This will leave the files around and clutter up developers machines. The point of using the context manager with a temp file is to prevent that. Why is this needed?

Reply:
Windows cannot handle `NamedTemporaryFile` being closed and reopened without it. The files are fractions of KB and will still be removed on reboot.

Reply:
bcb1959f8953f37714da61eaab03775ab9ab625d